tests/unit/lang/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 add_subdirectory(string_view_utf8)
 add_subdirectory(translator)
+add_subdirectory(eeprom)
```

Comment:
looks like the unit tests are missing from this PR

Reply:
Missing unit tests added